brd-ios/breadwallet/src/Helpers/PickerViewViewController.swift

```diff
@@ -146,7 +146,14 @@ extension PickerViewViewController: UIPickerViewDataSource, UIPickerViewDelegate
     }
     
     public func pickerView(_ pickerView: UIPickerView, didSelectRow row: Int, inComponent component: Int) {
-        selection = Index(column: component, row: row)
+        guard pickerView.numberOfComponents == values.count else { return }
+        let components = 0..<pickerView.numberOfComponents
+        
+        if component == components.first {
+            selection.primaryRow = row
+        } else if component == components.last {
```

Comment:
could this be pickerView.numberOfComponents?

Reply:
has to be -1. line 150.